<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>y_zflcx</class>
 <widget class="QDockWidget" name="y_zflcx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>自费率查询</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>开始日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>结算日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit_2"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>随机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>随机样本</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>超级喵喵</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>科室</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>入院日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出院日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>总费用</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>西药费</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>西药自费部分</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
